liquidprompt

```diff
@@ -1254,51 +1254,59 @@ _lp_grep_fields() {
     done <"$file"
 }
 
-# Adds as much $3 character (e.g. spaces) between $1 and $2,
+# Adds as much $3 (fillchars) character(s) (e.g. spaces) between $1 (left) and $2 (right),
 # so as to make the resulting string the same width as the current terminal.
+# If $4 (splitends) is true, then will cut $3 (fillchars) so as to perfectly fill the available space.
+# WARNING: this function may be slow if you put escaped characters in $3 (fillchars)
+#          and the width of the gap to fill is large. Thus, any escaped character is
+#          removed from $3 (fillchars) at the beginning.
```

Comment:
This does not make much sense. If escape sequences are removed, how can it be slow in that case?